packages/opentelemetry-core/src/common/time.ts

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as types from '@opentelemetry/types';
+import { performance } from 'perf_hooks';
+
+const NANOSECOND_DIGITS = 9;
+const SECOND_TO_NANOSECONDS = Math.pow(10, NANOSECOND_DIGITS);
+
+// Converts a number to HrTime
+function numberToHrtime(time: number): types.HrTime {
```

Comment:
Based on the use below where `performance.timeOrigin` and `Date.getTime` are passed in here, I believe the units of `time` should be epoch milliseconds. Given that, what would you think about calling it `epochMillis` to be clearer?

Also, given that we need to split the number into `[seconds, nanos]`, I believe this needs a `/ 1000` step at the start to convert `epochMillis` into `epochSeconds`, and then into `wholeSeconds` and `wholeNanos` parts (variable names can be called different things).